{{cookiecutter.project_slug}}/requirements.txt

```diff
@@ -1,2 +1 @@
 canonicalwebteam.flask-base==0.3.3
-gunicorn[gevent]
```

Comment:
@jpmartinspt Why we don't need this library?

Reply:
flask-base install talisker and talisker brings gunicorn.